flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/rules/logical/PushProjectIntoTableSourceScanRule.scala

```diff
@@ -94,16 +94,12 @@ class PushProjectIntoTableSourceScanRule extends RelOptRule(
     }
 
     // project push down does not change the statistic, we can reuse origin statistic
-    val newTableSourceTable = new TableSourceTable(
+    val newTableSourceTable = tableSourceTable.copy(
       newTableSource,
-      tableSourceTable.isStreamingMode,
-      tableSourceTable.statistic,
-      Option(usedFields),
-      tableSourceTable.catalogTable)
+      tableSourceTable.getStatistic,
```

Comment:
no need to pass the statistics to this copy method

Reply:
Yep, thanks.